torch/_dynamo/utils.py

```diff
@@ -990,6 +990,10 @@ def fake_wrapper(e):
         return e
 
     def visit(n: torch.fx.Node):
+        if "example_value" not in n.meta:
+            unimplemented(
+                f"Unsupported arg/kwarg doesn't have node.meta['example_value']: {n.format_node()}"
+            )
```

Comment:
Offline synced with @davidberard98, the pattern we are trying to fix is: users access attribute which might not existed. The non-existent attribute actually has different meanings:
* It's true non-existent, e.g, the added unit test where users wrap the code with ```try...catch...```. 
* It's because of dynamo is lacking of support yet, just need time to add them.

I think ```unimplemented``` is okay for both cases. At mean time, we keep to add those missing features. Just not sure if it may cover up some errors in fake tensor propagation. Let's check with fake tensor folks.